.github/workflows/pythonapp.yml

```diff
@@ -176,7 +185,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: apt install
       run: |
-        if [ ${{ matrix.environment }} != "PT16+CUDA110" ]; then \
+        if [ ${{ matrix.environment }} != "PT16+CUDA110" ] && [ ${{ matrix.environment }} != "PT14+NV20.03" ]; then \
```

Comment:
May I know why add this logic?
Thanks.

Reply:
this is to skip the apt-get commands if it's a pytorch-based docker image.